src/tests/system/tests/test_ldap.py

```diff
@@ -14,173 +14,102 @@
 from sssd_test_framework.topology import KnownTopology
 
 
-@pytest.mark.ticket(bz=[795044, 1695574])
-@pytest.mark.importance("critical")
-@pytest.mark.parametrize("modify_mode", ["exop", "ldap_modify", "exop_force"])
-@pytest.mark.parametrize("use_ppolicy", ["true", "false"])
-@pytest.mark.parametrize("sssd_service_user", ("root", "sssd"))
 @pytest.mark.topology(KnownTopology.LDAP)
-@pytest.mark.require(
-    lambda client, sssd_service_user: ((sssd_service_user == "root") or client.features["non-privileged"]),
-    "SSSD was built without support for running under non-root",
-)
-@pytest.mark.builtwith("ldap_use_ppolicy")
-def test_ldap__password_change_using_ppolicy(
-    client: Client, ldap: LDAP, modify_mode: str, use_ppolicy: str, sssd_service_user: str
+@pytest.mark.parametrize("modify_mode", ["exop", "ldap_modify", "exop_force"])
+@pytest.mark.importance("critical")
+def test_ldap__ppolicy_change_password(
+    client: Client,
+    ldap: LDAP,
+    modify_mode: str
 ):
     """
-    :title: Password change using ppolicy
-    :description: PPolicy overlay is the latest implementation of IETF password policy for LDAP.
-    This extends the password policy for the LDAP server and is configured in SSSD using
-    'ldap_use_ppolicy'.
-
-    Two password modification modes are tested, Extended Operation (exop), the default and then
-    LDAP (ldapmodify), set by 'ldap_pwmodify_mode' parameter.
-    :note: This feature is introduced in SSSD 2.10.0
+    :title: User logins and issues a password change
```

Comment:
Same comment as earlier about the title with "logins".  If you change the one in test_authentication.py, this one should be changed to match as well.

Reply:
Closing this one as resolved to match the other.   No changes needed here.